### PR TITLE
You can no longer build robots on Castlestation

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Kept you waiting, huh?
As I mentioned in #28545 I was gonna work on this, and now Castletation's Will prevents people from building AI and Cyborgs. After all, why would you WANT machines if they're the bread and butter of tyranny?

robots

:cl:
 * tweak: Castlestation will now prevent you from creating AI and Cyborgs. Unholy magics that mess with the fabric of reality are unaffected.

### CODE REVIEW

**Comment by PlausibleDeniability** [comment]
Nice, should remove the posibrain from robotics too

**Comment by Dacendeth** [comment]
if you can still build mommis posibrains still have a purpose

**Comment by boy2mantwicethefam** [comment]
MoMMIs have the H-word (human) pass

**Comment by DamianX** [comment]
I really, really, really like this image. Do you mind if I save it?

**Comment by boy2mantwicethefam** [comment]
Me too! Save it, it's all yours my friend 😃

**Comment by jknpj** [comment]
Who is "The station" ?

File: code/game/machinery/computer/ai_core.dm [hunk 1]
```diff
@@ -101,6 +101,9 @@
 				if(!prison.brainmob)
 					to_chat(user, "<span class='warning'>Sticking an empty [P] into the frame would sort of defeat the purpose.</span>")
 					return
+				if(!map.map_specific_conditions("robot check"))
```
Comment (Kurfursten) [inline code/game/machinery/computer/ai_core.dm]: use defines

**Review by Kurfursten — CHANGES_REQUESTED** [review]

**Review by Kurfursten — CHANGES_REQUESTED** [review]
once you use defines this looks fine

**Pull request merged**